<launch>
    <!-- 加载配置文件 -->
    <rosparam command="load" file="$(find bag_converter)/config/extrinsics.yaml" />
    
    <!-- 启动转换节点 -->
    <node name="dual_bag_to_cloud_info_converter" pkg="bag_converter" type="bag_to_cloud_info_node" output="screen" />
    
    <!-- 启动 RViz 可视化 -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find bag_converter)/rviz/accumulation_visualization.rviz" />
</launch>
